flash.py

```diff
@@ -0,0 +1,4 @@
+import os
+
+#This Should work correctly, if not let me know -Sam
+os.system("avrdude -c usbasp -p t84 -B 20 -U flash:w:target/avr-attiny84/release/semasher.elf")
```

Comment:
I'd like it if you made the parameters to this function programmable.

Something like:
```python3
os.system(f"avrdude -c {PROGRAMMER} -p {TARGET} -B {TARGET_CLOCK_MHZ} -U flash:w:{BIN}")
```